addons/binding/org.openhab.binding.exec/ESH-INF/thing/thing-types.xml

```diff
@@ -29,7 +29,7 @@
             <parameter name="interval" type="integer"  required="false">
                 <label>Interval</label>
                 <description>Interval, in seconds, the command will be repeatedly executed</description>
-                <default>60</default>
+                <default>0</default>
```

Comment:
Should be possible of course, but I'm not sure if the binding is tested with a null value, could be a potential cause of NullPointerException. That's why I choosed the "secure" way to set it to 0 making sure not breaking anything.